src/components/map.js

```diff
@@ -1,11 +1,12 @@
-import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
-import * as L from "leaflet";
-import "leaflet/dist/leaflet.css";
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import * as L from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+import Version from './version';
 
 function Map({ events }) {
   return (
     <MapContainer
-      style={{ height: "100vh" }}
+      style={{ height: '100vh' }}
```

Comment:
hmm is it possible to only change the lines you require and leave everything else as it is for the moment?

We should add these style configs to the project, that will be in another issue / pull request

Reply:
Done.